src/cascadia/TerminalApp/defaults.json

```diff
@@ -277,6 +277,7 @@
         { "command": "closeWindow", "keys": "alt+f4" },
         { "command": "toggleFullscreen", "keys": "alt+enter" },
         { "command": "toggleFullscreen", "keys": "f11" },
+        { "command": "toggleBorderless" },

```

Comment:
Do we want to hold off on this one until command palette comes in? idk

Reply:
By "command palette comes in", I mean when we add everything to the defaults.json.

Reply:
Nah, I'm just gonna start slapping things into there as appropriate. Case in point: #6762